src/component/form/Input.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from "prop-types";
+
+import {POSITION} from "../../tool/constant";
+
+import Tooltip from "../ui/Tooltip";
+
+import "./input.scss";
+
+const Input = ({type, placeholder, hasError, onChange}) => {
+
+    const inputRequiredError = () => {
+        return (
+            <div className="form__input-error__image__frame">
```

Comment:
If you are following the BEM structure for your styling, this breaks those rules. It should always be `block__element--modifer`

What this looks like is `block_element_element_element`